### PR TITLE
Add WebNN Operators of ONNX Runtime Web

### PR DESCRIPTION
**Pull request by ibelem**
Thanks @Honry 's great work on WebNN EP of ONNX Runtime Web, please take a look if the following structure is OK or improvement needed. 

- [WebNN Option in ONNX Runtime Web](https://onnxruntime.ai/docs/tutorials/web/)
- [Import WebNN Experimental Feature](https://onnxruntime.ai/docs/get-started/with-javascript/web.html)
- [Build with WebNN support via JavaScript Execution Provider](https://onnxruntime.ai/docs/build/web.html)
- [WebNN Execution Provider](https://onnxruntime.ai/docs/tutorials/web/env-flags-and-session-options.html)
- [WebNN Operators](https://github.com/microsoft/onnxruntime/blob/main/js/web/docs/webnn-operators.md) - by Wanming Lin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@
 - [Import WebNN Experimental Feature](https://onnxruntime.ai/docs/get-started/with-javascript/web.html)
 - [Build with WebNN support via JavaScript Execution Provider](https://onnxruntime.ai/docs/build/web.html)
 - [WebNN Execution Provider](https://onnxruntime.ai/docs/tutorials/web/env-flags-and-session-options.html)
```
Comment (Honry) [inline README.md]: Suggest change to:

- [ONNXRuntime Web Tutorials](https://onnxruntime.ai/docs/tutorials/web/)
- [Build ONNXRuntime Web with WebNN Support](https://onnxruntime.ai/docs/build/web.html)
- [WebNN Operators Support Table](https://github.com/microsoft/onnxruntime/blob/main/js/web/docs/webnn-operators.md) - by Wanming Lin

Comment (Honry) [inline README.md]: Suggest change to:

- [ONNXRuntime Web Tutorials](https://onnxruntime.ai/docs/tutorials/web/)
- [Build ONNXRuntime Web with WebNN Support](https://onnxruntime.ai/docs/build/web.html)
- [WebNN Operators Support Table](https://github.com/microsoft/onnxruntime/blob/main/js/web/docs/webnn-operators.md) - by Wanming Lin

**Review by Honry — APPROVED** [review]
LGTM, thanks!

File: README.md (hunk 1 above)
Comment (ibelem) [reply README.md]: Fixed

Comment (ibelem) [reply README.md]: Fixed

**Pull request merged**